chatbot/botconfig.py

```diff
@@ -8,8 +8,3 @@
     "#bottest:cclub.cs.wmich.edu",
     "#ccawmunity:cclub.cs.wmich.edu"
 ]
-
-# if you want to stop typing your password every time you start the bot,
-# store the password in an environment variable.
-# run: export BOTPASSWORD=passw0rd in a terminal.
-password_env_variable = "BOTPASSWORD"
```

Comment:
maybe `password_env_variable_name`
either way I feel we should keep the more sensitive auth creds separated into their own .gitignored file